### ISSUE TITLE
[Request] - Duplicate Folder

### ISSUE THREAD

**Issue by 6lv1-blr** [eldertek/duplicatefinder#115]
I would like to filter folder that are an exact copy of another one.
This could avoid repetive duplicate file management.

Merci encore

**Comment by eldertek** [comment]
No, sorry, the development of such a feature is not planned, but if you have different suggestions while keeping the same idea in mind, we can consider them!
